yt_dlp/__pyinstaller/hook-yt_dlp.py

```diff
@@ -23,7 +23,7 @@ def get_hidden_imports():
     yield pycryptodome_module()
     yield from collect_submodules('websockets')
     # These are auto-detected, but explicitly add them just in case
-    yield from ('mutagen', 'brotli', 'certifi')
+    yield from ('mutagen', 'brotli', 'certifi', 'requests', 'urllib3')
```

Comment:
might be a good idea to collect submodules just in case:
- if `requests` or `urllib3` have any hidden imports of their own they will be missed by pyinstaller without this
- if new version of pyinst changes something
- if user is building with older version of pyinst
```py
    # Only `websockets` is required, others are collected just in case
    for module in ('websockets', 'requests', 'urllib3'):
        yield from collect_submodules(module)
    # These are auto-detected, but explicitly add them just in case
    yield from ('mutagen', 'brotli', 'certifi')
```